sci-libs/vtk/vtk-9.1.0-r3.ebuild

```diff
@@ -180,6 +190,14 @@ pkg_pretend() {
 		CHECKREQS_DISK_BUILD="14G"
 	fi
 
+	if use qt6; then
+		ewarn "Qt6 is currently masked in ::gentoo and might not work or build"
+		ewarn "properly. You have been warned!"
```

Comment:
Can't say I like this message as we have to remove it ASAP when qt6 is unmasked. I'd just print it's experimental, if anything at all. People who want qt6 will need to manually unmask it, so they know it's experimental.

Reply:
I can remove this message completely if be. Just thought it would be better to print a notification just in case.